server/index.js

```diff
@@ -53,4 +53,11 @@ app.put('/reviews', (req, res) => {
     .catch(err => console.log('there was an err finding in db'));
 });
 
+app.delete('/delete/reviews',(req, res) => {
+  let query = {_id: req.body._id};
+  Reviews.remove(query)
+    .then(data => res.send('Successfully deleted!'))
+    .catch(err => console.log('Error deleting review'))
```

Comment:
You will want to `console.log` the `err` param so you know what's going on.

Reply:
Good call, made the change!